ui/app/AppLayouts/Profile/views/RPCStatsModal.qml

```diff
@@ -0,0 +1,163 @@
+            id: searchBox
+
+            Layout.fillWidth: true
+            Layout.bottomMargin: 16
+        }
+
+        StatusListView {
+            id: resultsListView
+
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            spacing: 2
+
+            ListModel {
+                id: sourceModel
+            }
+
+            header: StatusListItem {
+                title: qsTr("Total")
+                statusListItemTitle.customColor: Theme.palette.directColor1
+                statusListItemLabel.customColor: Theme.palette.directColor1
+                label: d.totalFilteredCalls + " of " + d.totalCalls
+                enabled: false
+                z: 3 // Above delegate (z=1) and above section.delegate (z = 2)
+            }
+            headerPositioning: ListView.OverlayHeader
+
+            model: SortFilterProxyModel {
+                sourceModel: sourceModel
+                sorters: RoleSorter {
+                    roleName: "count"
+                    sortOrder: Qt.DescendingOrder
+                }
+
+                filters: FastExpressionFilter {
+                    function spellingTolerantSearch(data, searchKeyword) {
+                        const regex = new RegExp(searchKeyword.split('').join('.{0,1}'), 'i')
+                        return regex.test(data)
+                    }
+
+                    enabled: !!searchBox.text
+                    expression: {
+                        searchBox.text
+                        let keyword = searchBox.text.trim().toUpperCase()
+                        return spellingTolerantSearch(model.method, keyword)
+                    }
+                    expectedRoles: ["method"]
+                }
+            }
+
+            SumAggregator {
+                id: countAggregator
+
+                model: resultsListView.model
+                roleName: "count"
+            }
+
+            delegate: StatusListItem {
+                title: model.method
+                label: model.count
+                enabled: false
+            }
+
+            Component.onCompleted: {
+                d.updateModel()
+            }
+        }
+    }
+
+    footer: StatusDialogFooter {
+        leftButtons: ObjectModel {
+            StatusButton {
+                text: "Refresh"
+                onClicked: {
+                    d.updateModel()
+                }
+            }
+            StatusButton {
+                text: "Reset"
+                onClicked: {
+                    root.walletStore.resetRpcStats()
+                    d.updateModel()
+                }
+            }
+        }
+
+        rightButtons: ObjectModel {
+            CopyToClipBoardButton {
+                id: copyToClipboardButton
+
+                onCopyClicked: root.walletStore.copyToClipboard(textToCopy)
+                onPressed: function() {
+                    let copiedText = "Total" + '\t' + d.totalFilteredCalls + " of " + d.totalCalls + '\n' + '\n'
```

Comment:
qsTr() here too

Reply:
That is purely for development/debugging, so it might be just fine to not translate those parts of the app.

Reply:
Yeah I'm fine with either approach, doesn't have to be strictly translatable